controllers/api-controller.js

```diff
@@ -0,0 +1,14 @@
+const fs = require("fs/promises");
+
+exports.getEndpoints = (request, response, next) =>{
+    fs.readFile("./endpoints.json", "utf-8").then((data)=>{
+        const endpoints = JSON.parse(data)
```

Comment:
This is fine as it is, but you could simply require in the endpoints.json file and send it, rather than having to use readfile 👍